lib/Styles/Common/_borders.scss

```
@@ -0,0 +1,13 @@
+@mixin border-interactive {
+    box-shadow:
+        outset-highlight("full"),
+        outset-shadow(1);
+
+    border-style: solid;
+    border-color: $border-color;
+    border-width: 1px;
+}
+
+@mixin border-interactive-roundrect {
+    border-radius: 3px;
```

Let's scale this on window radius for our nested radii theory
```suggestion
    border-radius: rem($window_radius / 3);
```